static/js/stream_popover.js

```diff
@@ -588,11 +590,8 @@ export function register_click_handlers() {
 export function register_stream_handlers() {
     // Stream settings
     $("body").on("click", ".open_stream_settings", (e) => {
-        const sub = stream_popover_sub(e);
         hide_stream_popover();
-
-        const stream_edit_hash = hash_util.stream_edit_url(sub);
-        browser_history.go_to_location(stream_edit_hash);
+        e.stopPropagation();
```

Comment:
Can you explain why we're using `e.stopPropagation` here? In any case, I think a comment is warranted explaining we're intentionally falling through to let the browser change the URL fragment so that navigates for us.